xml/deployment_installation.xml

```diff
@@ -715,26 +741,352 @@
+     <sect5>
+      <title>Configuration Files</title>
+      <para>
+       There are two separate configuration files: <literal>user-data</literal>
+       and <literal>meta-data</literal>. Each node needs both. Thus, you must at
+       a minimum prepare one pair of files for the &admin_node;, and another
+       pair of files for all the &worker_node;s.
+      </para>
+      <para>
+       The two files must be placed into a subdirectory. For the &admin_node;,
+       this must be called <literal>cc-admin</literal>. For the &worker_node;s,
+       this must be called <literal>cc-worker</literal>.
+      </para>
+      <para>
+       So, for instance, if your working directory is called <literal>~/cloud-config</literal>,
+       for the admin node, you need:
+      </para>
+      <screen>~/cloud-config/cc-admin/user-data</screen>
+      <para>
+       and
+      </para>
+      <screen>~/cloud-config/cc-admin/meta-data</screen>
+      <para>
+       For a &worker_node;, you need:
+      </para>
+      <screen>~/cloud-config/cc-worker/user-data</screen>
+      <para>
+       and
+      </para>
+      <screen>~/cloud-config/cc-worker/meta-data</screen>
+      <para>
+       The same <literal>meta-data</literal> file can be used for both node types.
+       Here is an sample <literal>meta-data</literal> file:
+       </para>
+      <screen>
+# File: cloud-config/cc-worker/meta-data
+instance-id: <replaceable>iid-CAAS01</replaceable>
+network-interfaces: |
+  auto <replaceable>eth0</replaceable>
+  iface <replaceable>eth0</replaceable> inet dhcp
+       </screen>
+      <para>
+       The <literal>user-data</literal> file contains settings such as time
+       servers, the &rootuser; password, and the node type.
+      </para>
+      <para>
+       Here is an example <literal>user-data</literal> file for an &admin_node;:
+      </para>
+      <screen>
+# File: cloud-config/cc-admin/user-data
+#cloud-config
+debug: True
+disable_root: False
+ssh_pwauth: True
+chpasswd:
+  list: |
+    root:<replaceable>root</replaceable>
+    expire: False
+ntp:
+  servers:
+    - <replaceable>ntp1.example.com</replaceable>
+    - <replaceable>ntp2.example.com</replaceable>
+runcmd:
+  - /usr/bin/systemctl enable --now ntpd
+suse_caasp:
+  role: <replaceable>admin</replaceable>
+       </screen>
+      <para>
+       Here is an example for a &worker_node;:
+      </para>
+      <screen>
+# File cloud-config/cc-worker/user-data
+#cloud-config
+debug: True
+disable_root: False
+ssh_pwauth: True
+chpasswd:
+  list: |
+    root:<replaceable>root</replaceable>
+    expire: False
+suse_caasp:
+   role: <replaceable>cluster</replaceable>
+   admin_node: <replaceable>caas-admin.example.com</replaceable>
+       </screen>
+     </sect5>
+     <sect5>
+      <title>Preparing an ISO image</title>
+      <para>
+       Once you have edited the configuration files with your desired settings,
+       you must create an ISO image with the volume lable <literal>cidata</literal>
```

Comment:
label

Reply:
I can't even claim that is UK English, because it isn't. Fixed.